.github/scripts/release.py

```diff
@@ -37,8 +37,12 @@ def build_package(pkg_name):
         'README.md'
     ]
 
+    ignore_pattern = shutil.ignore_patterns()
+
     for dir in dirs_to_copy:
-        shutil.copytree(os.path.join(xmc_ino_root_path, dir), os.path.join(pkg_build_path, dir))
```

Comment:
Increase the version of the script to 0.2.2?

Reply:
The current version in master is 0.2.0. So 0.2.1 should be correct 🥲 But I don't know why the git log is so strange. We revert this file some times a go....
But in final merge you will see the script is correctly versioned

Reply:
if that is the case 👍